tests/end_to_end/conftest.py

```diff
@@ -195,3 +196,48 @@ def relayers(worker_id):
     except ValueError:
         logger.info(f"Error while setting relayer index to {worker_id}")
     return
+
+
+# Uniswap fixtures
```

Comment:
@ClementWalter would like your view on this: do you find this ok to import uniswap fixtures in the end_to_end conftest and delete the uniswap conftest ?
It is necessary as a test with DualTokenVM is using UniswapRouter to ensure adding liquidity and swap works.